pages/projects.tsx

```diff
@@ -35,14 +41,22 @@ function Projects({ projects, githubColors }: ProjectsProps): JSX.Element {
           a (work-in-progress) heads-up overview of the projects that power ACM at UCLA.
         </p>
         <hr />
+        <SearchFilter
+          projects={projects}
+          setFilteredProjects={setFilteredProjects}
+        />
+        <hr/>
         <div className="row same-height-grid">
-          {projects.map((project, i) => {
-            return (
-              <div className="col-4" key={project.name}>
-                <ProjectCard project={project} vertical preload={i < 3} githubColors={githubColors} />
-              </div>
-            );
-          })}
+          {filteredProjects.length > 0
+            ? filteredProjects.map((project, i) => {
+              return (
+                <div className="col-4" key={project.name}>
+                  <ProjectCard project={project} vertical preload={i < 3} githubColors={githubColors} />
+                </div>
+              );
+            })
+            : <h1>No results found</h1>
```

Comment:
As a general rule, we only have a single h1 on a page at any given time. Subheadings can be `h2` and it's fine to have multiple `h2` tags present on a page at once!